lms/static/js/dashboard/legacy.js

```diff
@@ -116,11 +116,8 @@
              $('#failed-verification-banner').addClass('is-hidden');
          });
 
-         $('#upgrade-to-verified').click(function(event) {
-             var user = $(event.target).closest('.action-upgrade').data('user'),
-                 course = $(event.target).closest('.action-upgrade').data('course-id');
-
-             Logger.log('edx.course.enrollment.upgrade.clicked', [user, course], {location: 'learner_dashboard'});
+         $('.action-upgrade').click(function(event) {
```

Comment:
@dianakhuang: The main fix was to use the class `.action-upgrade` to get this event for all clicks, not just clicks to the text on the button.